auraed/src/bin/main.rs

```diff
@@ -29,6 +29,34 @@
 \* -------------------------------------------------------------------------- */
 
 #![warn(clippy::unwrap_used)]
+#![deny(bad_style,
+        const_err,
```

Comment:
This is [deny-by-default](https://doc.rust-lang.org/rustc/lints/listing/deny-by-default.html), so doesn't need to be included